projects/cdk/schematics/ng-add/tests/schematic-ng-add.spec.ts

```diff
@@ -254,13 +254,13 @@ describe('ng-add', () => {
 
         expect(tree.readContent('test/app/app.module.ts')).toBe(
             `import { NgDompurifySanitizer } from "@taiga-ui/dompurify";
-import { TuiRoot, TuiDialogModule, TuiAlertModule, TUI_SANITIZER } from "@taiga-ui/core";
+import { TuiRoot, TuiDialog, TuiAlert, TUI_SANITIZER } from "@taiga-ui/core";
 import { BrowserAnimationsModule } from "@angular/platform-browser/animations";
 import {NgModule} from '@angular/core';
 import {AppComponent} from './app.component';
 
 @NgModule({declarations: [AppComponent],
-    imports: [BrowserAnimationsModule, TuiRoot, TuiDialogModule, TuiAlertModule],
+    imports: [BrowserAnimationsModule, TuiRoot, TuiDialog, TuiAlert],
```

Comment:
We no longer need dialog and alert in main module

Reply:
fixed